kore/src/main/java/org/kframework/parser/KoreParser.java

```diff
@@ -39,7 +39,7 @@ public K parseString(String koreString) {
             Pattern kore = textToKore.parsePattern(koreString);
             return koreToK.apply(kore);
         } catch (ParseError parseError) {
-            throw KEMException.criticalError("Parse error" );
+            throw KEMException.criticalError("Parse error.\n" + parseError.msg() );
```

Comment:
This should probably chain the exception also.

Reply:
It's actually better this way; and it doesn't need the message anymore.